robolectric/src/main/java/org/robolectric/util/ActivityController.java

```diff
@@ -178,6 +188,22 @@ public void run() {
       }
     });
 
+    ViewRootImpl root = component.getWindow().getDecorView().getViewRootImpl();
+    ShadowViewRootImpl shadow = (ShadowViewRootImpl) ShadowExtractor.extract(root);
+    Display display = Shadow.newInstanceOf(Display.class);
+    ShadowDisplay shadowDisplay = Shadows.shadowOf(display);
+    Rect frame = new Rect(0, 0, shadowDisplay.getWidth(), shadowDisplay.getHeight());
+    Rect insets = new Rect(0, 0, 0, 0);
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
One pattern already in place is:-
https://github.com/robolectric/robolectric/blob/ed57c3393646ca79bc69d4958feeca583c0850a6/robolectric/src/main/java/org/robolectric/internal/runtime/RuntimeAdapterFactory.java

Could add a method callViewRootImplDispatchResized - @erd - thoughts?